src/dos/program_loadrom.cpp

```diff
@@ -42,13 +42,14 @@ void LOADROM::Run(void) {
     }
     uint8_t drive;
     char fullname[DOS_PATHLENGTH];
-    localDrive* ldp=0;
     if (!DOS_MakeName((char *)temp_line.c_str(),fullname,&drive)) return;
 
     try {
         /* try to read ROM file into buffer */
-        ldp=dynamic_cast<localDrive*>(Drives[drive]);
-        if (!ldp) return;
+	const auto ldp = dynamic_cast<localDrive*>(Drives.at(drive));
+	if (!ldp) {
+		return;
+		}
```

Comment:
wonky formatting

Reply:
ahh good catch.. indeed: this file is tangled up with a combination of tabs and spaces :facepalm:
Will format it in a separate commit.